src/main/python/camdkit/model.py

```diff
@@ -36,6 +35,7 @@ class CaptureFPS(StrictlyPositiveRationalParameter):
   canonical_name = "captureRate"
   sampling = Sampling.STATIC
   units = "hertz"
+  section = "camera"
```

Comment:
Doesn't a tracking capture also happen over a duration? If `duration` is not available for tracking metadata, how is the sampling frequency of each metadata parameter determined?

If tracking metadata is not necessarily sampled regularly, and instead each sample contains a timestamp, then we should probably use something other than `ParameterContainer`.